### PR TITLE
refactor: ethpm-types checksum

### PR DESCRIPTION
**Pull request by NotPeopling2day**
### What I did

Refactor for breaking changes after `ethpm-types` merge,  addressing `checksum` processing.

fixes: #

### Checklist

- [ ] All changes are completed
- [ ] New test cases have been added
- [ ] Documentation has been updated

### CODE REVIEW

**Comment by NotPeopling2day** [comment]
Requires the merge of https://github.com/ApeWorX/ethpm-types/pull/17

File: src/ape/managers/project.py [hunk 1]
```diff
@@ -112,13 +112,12 @@ def file_needs_compiling(source: Path) -> bool:
                 return True  # New file added
 
             cached = cached_sources[path]
-            cached.compute_checksum(algorithm="md5")
+            cached.calculate_checksum()
```
Comment (unparalleled-js) [inline src/ape/managers/project.py:115]: I think this returns the checksum now or something (as opposed to setting it)?

**Review by unparalleled-js — COMMENTED** [review]
Before merging this, we will need to release ethpm-types and bump the version in setup.py

File: src/ape/managers/project.py (hunk 1 above)
Comment (fubuloubu) [reply src/ape/managers/project.py:115]: yes this

**Comment by fubuloubu** [comment]
> Before merging this, we will need to release ethpm-types and bump the version in setup.py

Problem here is that as soon as we release `ethpm-types`, then ape core breaks because we never pinned the top of that in `setup.py`

I think this might need to wait for `v0.2.0` or something, and we can sneak in the `ethpm-types` pinning now (in a separate PR) to prepare for that

File: src/ape/managers/project.py (hunk 1 above)
Comment (NotPeopling2day) [reply src/ape/managers/project.py:115]: Yeah, I'll go ahead and close this PR and open a pinning PR in its place.

**Comment by NotPeopling2day** [comment]
Closing in favor of pinning and fixing breaking changes later.

**Pull request closed without merging**